src/Body.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+function Body() {
+    return (
+      <div className="flex flex-col sm:flex-row-reverse items-start h-screen pt-10 min-w-[456px] ml-4 mr-4">
+
+        {/*Popular Tag pane on the right side will move to top on smaller view*/}
+        <div className="w-full sm:w-[35%] sm:ml-4">
```

Comment:
Popular tag should be its own component so it can be reused.

Reply:
Addressed. Also changed the naming to be more future-proof. Please see my comments on my code. Thanks!